backend/isoko-backend/src/handlers/business/put-edit-business-page.js

```diff
@@ -1,3 +1,54 @@
+const dynamodb = require('aws-sdk/clients/dynamodb');
+const _ = require('lodash');
+const docClient = new dynamodb.DocumentClient();
+const { BUSINESS_TABLE } = require('../../constants');
+
+/**
+ * 
+ * @param {*} name of complex object 
+ * @param {*} obj complex object
+ * @param {*} exprs array that contains pieces of UpdateExpression 
+ * @param {*} attrValues object that contains ExpressionAttributeValues object
+ * @param {*} count int that keeps track of current mapping 
+ */
+const buildComplexObjectExpression = (objName, obj, exprs, attrValues, count) => {
+   const objKeys = _.keys(obj)
+   objKeys.forEach((k) => {
+      const mapping = `:${String.fromCharCode(count)}`; 
+      const updateName = `${objName}.${k}`; 
+      exprs.push(`${updateName} = ${mapping}`); 
+      attrValues[mapping] = _.get(obj, k); 
+      count += 1; 
```

Comment:
This increment to count won't actually get applied to the variable count used in buildUpdateExpression since JS passes primitives by value and not reference. I left a comment on the test file which would show the bug in the update expression.

Reply:
If you wanted to fix this you could return the updated count from this function since it doesn't return anything atm

Reply:
i had a feeling js did some weird stuff like that smh... ty